projects/storefrontlib/src/lib/my-account/order/order-history/order-history.component.ts

```diff
@@ -71,15 +72,15 @@ export class OrderHistoryComponent implements OnInit, OnDestroy {
     this.fetchOrders(event);
   }
 
-  pageChange(page: number) {
+  pageChange(page: number): void {
     const event = {
       sortCode: this.sortType,
       currentPage: page
     };
     this.fetchOrders(event);
   }
 
-  goToOrderDetail(order) {
+  goToOrderDetail(order): void {
     this.routing.go(['my-account/orders/', order.code]);
   }
 
```

Comment:
The `fetchOrders()` can be `void`, right?

Reply:
Yuppers, thanks